python/neuroglancer/write_annotations.py

```diff
@@ -197,20 +368,31 @@ def _add_obj(self, coords: Sequence[float], id: Optional[int], **kwargs):
             id=id, encoded=encoded.tobytes(), relationships=related_ids
         )
 
+        chunk_index = self.get_chunk_index(np.array(coords[: self.rank]))
```

Comment:
For lines and bounding boxes, we need to add it to all intersecting chunks, which may not be a single chunk.

Reply:
ah..good catch.  I had to add some extra arguments to handle this because I think ellipsoids probably we don't want to use all the coordinates since the second set of coordinates are related to their radii.  The class was missing an implementation for ellipsoids anyway so i've added that now.

let me know what you think of that fix.